wharehauora-wifi-gateway/wharehauora-wifi-gateway.ino

```diff
@@ -2,25 +2,45 @@
 #include <SPI.h>
 #include <WiFiManager.h>         //https://github.com/tzapu/WiFiManager
 #include <ESP8266WiFi.h>
+#include <Ticker.h>
+
+#define FIRMWAREVERSION "1"
```

Comment:
@Br3nda I don't know how you want to do versioning but I am not really a fan of any sort of semantics in versions, I prefer to just jam a number there and increment away.

If you think a "major"."minor"."point release" versioning scheme is better, I'll change it :)